### ISSUE TITLE
integration broken since last update un my HA instance

### ISSUE THREAD

**Issue by arnalll** [oziee/ha-solcast-solar#235]
### The problem

since last update integration is broken i n my instance, was working fine but with last update it throwed me an error, cant remember exactly what  the issue was at that time , then i tried to fix the issue by deleting the integration and re-downloading it. Now i cant even configure the integration, HA throw me this : "couldnt load configuration flow : 500 Internal Server Error Server got itself in trouble". I ve tried with previous releases but i m stuck at this point.

### What version of Solcast Integration has the issue?

v4.0.13

### What version of Home Assistant are you using?

Home Assistant 2023.7.2

### Logs debug/info/error output

```txt
raise RequirementsNotFound(name, list(failures))
homeassistant.requirements.RequirementsNotFound: Requirements for solcast_solar not found: ['aiohttp>=3.8.5'].
```

### Additional information

ive tried to update aiohttp and it appears to be 3.8.5... don't know if you can help me.

thanks in advance.

best regards, awesome work

**Comment by oziee** [comment]
hmm.. in HACS when installing select an older version like 4.0.12 to install.. that uses 3.6.2

**Comment by arnalll** [comment]
Same issue with 4.0.12 ...

**Comment by oziee** [comment]
did you completely uninstall the integration / restart HA / reinstall the integration ?

**Comment by arnalll** [comment]
yes i did

**Comment by oziee** [comment]
dunno then

**Comment by arnalll** [comment]
Logger: homeassistant.util.package
Source: util/package.py:107
First occurred: 15:39:55 (3 occurrences)
Last logged: 15:40:05

Unable to install package aiohttp>=3.8.5: ERROR: Cannot install aiohttp>=3.8.5 because these package versions have conflicting dependencies. ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts [notice] A new release of pip is available: 23.1.2 -> 23.3.1 [notice] To update, run: pip install --upgrade pip   

well gonna see what i can do with these log info.

thanks a lot

**Comment by oziee** [comment]
Unfortunately, Home Assistant controls all. The manifest file List dependencies that need to be installed in a HA controls all this.

Do you have a backup file that you can reinstall from just before updating?
